src/actions/index.js

```diff
@@ -0,0 +1,17 @@
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const defaultBook = {
+  id: '',
+  title: '',
+  category: '',
+};
+export const createBook = (book = defaultBook) => ({
+  type: ADD_BOOK,
+  book,
+});
+
+export const removeBook = ({ id }) => ({
+  type: REMOVE_BOOK,
+  id,
+});
```

Comment:
- [ ] The `REMOVE_BOOK` action should use the **book object** as an argument, not the id.
Screenshot from pathwright 👇🏾 
<img width="783" alt="Screen Shot 2021-04-15 at 9 56 17 PM" src="https://user-images.githubusercontent.com/48804258/114930529-692dda00-9e35-11eb-8c49-9fd75983419b.png">

Reply:
👍